.pre-commit-config.yaml

```diff
@@ -23,7 +23,7 @@ repos:
   - id: pytest
     name: pytest
     entry: uv
-    args: [run, --all-extras, pytest, -n, auto, tests/ert/unit_tests, tests/everest, --hypothesis-profile=fast, -m, not integration_test]
+    args: [run, --all-extras, pytest, -n, auto, tests/ert/unit_tests, tests/everest, --hypothesis-profile=fast, --dist, loadgroup, -m, not integration_test]
```

Comment:
We should probably just combine the two and use `args = ["run" "--all-extras", "just", "raid-tests"]` and remove `nice` from `rapid-tests` (you can always use `nice just rapid-tests` if you want that behavior.